google/auth/jwt.py

```diff
@@ -472,32 +446,3 @@ def signer_email(self):
     @_helpers.copy_docstring(credentials.Signing)
     def signer(self):
         return self._signer
-
-    def before_request(self, request, method, url, headers):
```

Comment:
Because once the one-time token logic is removed, it's identical to the superclass's `before_request`:

```
if not self.valid():
    self.refresh(request)
self.apply(headers)
```